### DISCUSSION TITLE
debug.log returns nothing but warning messages

### DISCUSSION THREAD

**Discussion by qbol69** [Chia-Network/chia-blockchain#6067] [Support]
So I have some plots set up, node is synchronized, in gui all of the plots are with status "Farming", although debug file is full of same repetitive error msg's:

`2021-05-26T14:57:09.149 full_node chia.rpc.rpc_server     : WARNING  Sending data failed. Exception Traceback (most recent call last):
  File "chia\rpc\rpc_server.py", line 67, in _state_changed
  File "aiohttp\client_ws.py", line 150, in send_str
  File "aiohttp\http_websocket.py", line 687, in send
  File "aiohttp\http_websocket.py", line 656, in _send_frame
  File "aiohttp\base_protocol.py", line 84, in _drain_helper
AssertionError
.`

Is this something I should be worried about? I've seen in some videos that this debug log should contain information about the validations, how much time it takes for attempts and stuff like that, but I don't see such things there, just the error notification listed above.

**Comment by awakefulsleep373** [comment]
Same here.  300 plots so far, no reward yet.  Will keep you posted.

**Comment by emlowe** [comment]
You may need to set your debug level to INFO to see more .. well.. info.

**Reply by awakefulsleep373** [reply]
Thanks so much!!  Just made the change and saved the file.  Will close and restart and check after plots finish. 👍🙏
